oe_theme.theme

```diff
@@ -2063,21 +2063,10 @@ function oe_theme_preprocess_paragraph__oe_carousel(array &$variables): void {
       $cacheability->applyTo($variables);
       continue;
     }
-    $field_name = $source->getConfiguration()['source_field'];
-    if ($source instanceof Image && ($file_entity = $media->get($field_name)->entity)) {
-      $uri = $file_entity->getFileUri();
-      $cacheability->addCacheableDependency($file_entity);
-    }
-    elseif ($source instanceof MediaAvPortalSourceInterface) {
-      $resource_ref = $media->get($field_name)->value;
-      $uri = 'avportal://' . $resource_ref . '.jpg';
-    }
-    // The uri might be empty if the source is of type Image and the file entity
-    // was deleted.
-    if (empty($uri)) {
-      $cacheability->applyTo($variables);
-      continue;
-    }
+    $thumbnail = $media->get('thumbnail')->first();
```

Comment:
Why are we using the thumbnail all of a sudden? We can just create the styled image by loading it and using the $uri we already generated no?
    $style = \Drupal::entityTypeManager()->getStorage('image_style')->load('oe_theme_full_width');
   $style->buildUrl($uri)